src/libstd/env.rs

```diff
@@ -599,24 +602,24 @@ pub fn temp_dir() -> PathBuf {
 /// Ok("/home/alex/foo")
 /// ```
 ///
-/// And you make a symbolic link of the program:
+/// And you make a hard link of the program:
 ///
 /// ```bash
 /// $ ln foo bar
 /// ```
 ///
-/// When you run it, you won't get the original executable, you'll get the
```

Comment:
Was the change from `'` to `’` deliberate? (Not sure what the convention is here).

Reply:
deliberate but not something i feel strongly about. figure the line is already changing so i'm not ruining the diff in any way, but i'm fine keeping as-is

Reply:
I don't mind much either way however it looks like `'` is used a lot more than `’` in the existing doc comments, so my personal vote would be to keep `'` for consistency (but feel free to keep this as-is to get a second opinion).

Reply:
`’` is the preferred symbol to use for apostrophe in english text according to the Unicode standard. `'` is more commonplace because people can’t always be bothered to type Unicode symbols.

My personal vote is to follow the preferences set forth by the standard (I myself try to type unicode `’`), but is not something I would bug other people about whether they do or don’t follow the practice.

Reply:
In all the docs i've written, i've just used the regular `'` because `’` is impossible to type on US-layout keyboards without a software compose key or other kind of software text replacement. On the other hand, the difference is likely too small to notice unless it was inconsistent within a single page. So here it's probably fine, but i wouldn't go using it on other pages without grabbing all the others on the same page like you did here. (And then possibly creating a tidy lint for it so we can keep on top of it. `>_>`)